src/pat/inject/inject.js

```diff
@@ -305,6 +314,7 @@ define([
             if ((cfg.history === "record") && ("pushState" in history)) {
                 history.pushState({'url': cfg.url}, "", cfg.url);
             }
+            $target.trigger("pat-inject-content-loaded-final");
         },
```

Comment:
The name of this event is misleading. The `pat-inject-content-loaded` event is triggered asynchronously after all images have loaded and might therefore be loaded after this event, so `-final` is not correct.

Why can you not use the existing `pat-inject-content-loaded` event?